javascrip.js

```diff
@@ -1,6 +1,6 @@
 function Cacique (rappi) {
     if (rappi === 'hamburguer') {
-        console.log('Xablau')
+        console.log('Xablau');
```

Comment:
Acho que poderia adicionar mais um console.log
```suggestion
        console.log('Xablau');
        console.log('Agora quero um KitKat');
```